chrome-extension/manifest.json

```
@@ -0,0 +1,27 @@
+{
+    "manifest_version": 2,
+    "name": "Fuck Off Twitter",
```

Can we keep the same wording and punctuation for consistency?

```suggestion
{
    "manifest_version": 2,
    "name": "Fuck off, @twitter",
```